From 1fc664c477f51bd58d655d9a3eb55ea6ee12e577 Mon Sep 17 00:00:00 2001
From: Fujii Hironori <Hironori.Fujii@sony.com>
Date: Sat, 18 Feb 2023 15:33:28 +0900
Subject: [PATCH 10/15] Fix cairo_matrix_transform_distance documentation

* Use dx and dy instead of dx1 and dy1 to match with the arguments
* Use dx_new and dy_new instead of dx2 and dy2 to match with other parts of the documentation
* Use xx, yx, xy, yy instead of a, b, c, d to match with other parts of the documentation
* Remove the last two sentences that doesn't make sense
---
 src/cairo-matrix.c | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/src/cairo-matrix.c b/src/cairo-matrix.c
index cb49adcbe..e8fd097ae 100644
--- a/src/cairo-matrix.c
+++ b/src/cairo-matrix.c
@@ -373,15 +373,10 @@ _cairo_matrix_multiply (cairo_matrix_t *r,
  * the returned vector is as follows:
  *
  * <programlisting>
- * dx2 = dx1 * a + dy1 * c;
- * dy2 = dx1 * b + dy1 * d;
+ * dx_new = xx * dx + xy * dy;
+ * dy_new = yx * dx + yy * dy;
  * </programlisting>
  *
- * Affine transformations are position invariant, so the same vector
- * always transforms to the same vector. If (@x1,@y1) transforms
- * to (@x2,@y2) then (@x1+@dx1,@y1+@dy1) will transform to
- * (@x1+@dx2,@y1+@dy2) for all values of @x1 and @x2.
- *
  * Since: 1.0
  **/
 void
-- 
2.34.1

